iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -52,6 +53,12 @@ static llvm::cl::opt<bool> clTraceDispatchTensors(
         "Trace runtime input/output tensors for each dispatch function."),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clEnable1x1ConvToMatmul(
```

Comment:
@ThomasRaoux this should be OK to enable by default right?

Reply:
It will regress performance if committed before https://github.com/google/iree/pull/5415 but it will improve performance if committed after.

Reply:
I think it's fine if we want to commit this first though

Reply:
I can submit this with default (off) and you can enable as part of 5415 ?

Reply:
Sure that works

Reply:
Done.